eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -85,34 +86,32 @@ vote_info =
   , stake_credential
   ]
 
-governance_procedure = 
-  [ voting_procedure
-  // proposal_procedure
+governance_procedure =
+  [ (0, voting_procedure)
+  // (1, proposal_procedure)
   ]
```

Comment:
This can be removed if we're storing votes and proposals in separate lists

Reply:
Done.